src/routes/metadata.ts

```diff
@@ -4,4 +4,34 @@ import * as db from '../tools/db';
 // for location, categories, etc
 
 export const register = (app: express.Application) => {
+
+    //GET ProductCategories
+    app.get('/api/v1/prodcats',(req, res) =>{
+        db.connection.query('SELECT * from ProductCategory;',(error,results)=>{
```

Comment:
```suggestion
        db.connection.query('SELECT * from ProductCategory;', (error, results) => {
```

Make sure that the spacing is like this for all endpoints